shiny/api-examples/data_frame/app-core.py

```diff
@@ -44,9 +44,11 @@ def summary_data():
 
     @reactive.calc
     def filtered_df():
-        # input.summary_data_selected_rows() is a tuple, so we must convert it to list,
+        req(summary_data.input_selected_rows())
+
+        # summary_data.selected_rows() is a tuple, so we must convert it to list,
         # as that's what Pandas requires for indexing.
-        selected_idx = list(req(input.summary_data_selected_rows()))
+        selected_idx = list(req(summary_data.input_selected_rows()))
```

Comment:
`req()` is done above. Removing here

```suggestion
        selected_idx = list(summary_data.input_selected_rows())
```